src/App.tsx

```diff
@@ -62,14 +63,14 @@ interface AppProps {
 
 function App({ walletReady }: AppProps) {
   const { initialRoute } = useApplicationSetup();
-
+  const { bottom } = useSafeAreaInsets();
   const handleNavigatorRef = useCallback((ref: NavigationContainerRef<RootStackParamList>) => {
     Navigation.setTopLevelNavigator(ref);
   }, []);
 
   return (
     <Portal>
-      <View style={sx.container}>
+      <View style={[sx.container, { paddingBottom: IS_ANDROID ? bottom : 0 }]}>
```

Comment:
This is necessary because some Android devices were going under 3 button navigation still. This should fix it.